auto_landing/include/auto_landing/tracking.hpp

```diff
@@ -0,0 +1,34 @@
+#include<ros/ros.h>
+#include<geometry_msgs/Twist.h>
+#include<nav_msgs/Odometry.h>
+#include<geometry_msgs/PoseStamped.h>
+#include<std_srvs/Trigger.h>
+
+namespace ariitk::auto_landing{
+    
+    class tracking {
```

Comment:
remove this indentation. namespace members are not indented.

Reply:
Also follow naming convention for classes (tracking -> Tracking)